spec/forms/publish/provider_contact_form_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+require_relative '../shared_examples/blank_validation'
```

Comment:
Do we need this?

Reply:
Not really, I just copied it over from 'support/provider_contact_form.rb`, it is marginally shorter. We might benefit from some shared examples for postcode, email, urn, ukprn types as a refactor at some point.

Reply:
no i meant, shouldn't it be auto loaded by default?

Reply:
No, I think the autoload would be files in `spec/support/shared_examples/`